README.md

```diff
@@ -1,2 +1,47 @@
-# ToDolist-React
-This project for the to do list with react libirary ,i used react events and component correctly this project.
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List React
+
+> This is the project of the to do list application. I set up the environment and tools needed to develop this app by a React.
+
+## Built With
+
+- javascript,react
+- github workflows
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
Your link demo is not working .please kindly provide the right link demo of the project
![code-review](https://user-images.githubusercontent.com/98626003/185052702-a5302c8f-ec78-4f39-903a-baa06f14af01.png)